src/cam_qhy.h

```diff
@@ -35,6 +35,9 @@
 #ifndef CAM_QHY_INCLUDED
 #define CAM_QHY_INCLUDED
 
+#define QHYCCD_OFF  0.0
+#define QHYCCD_ON   1.0
+

```

Comment:
let's move these definitions into cam_qhy.cpp as they do not need to be exposed here as part of the pure interface being defined in this file.

Reply:
Done